src/components/note-editor.js

```diff
@@ -0,0 +1,35 @@
+import { useNotes } from "../context/notes-context"
+import addNote from "../context/notes-functions/addNotes"
+import {useUser} from "../context/user-context"
+
+const NoteEditor = () => {
+
+    const {notesState : {title,body},notesDispatch} = useNotes()
+
+    const {userState,userDispatch} = useUser()
+
+    console.log(userState)
+
+    return(
+        <div className="half-width flex flex-column gap-s">
+            <input type = "text" id = "demo" className = "input-field" placeholder="Please Enter Title of your Note" style={{width : "100%"}} value = {title} onChange = {(e) => notesDispatch({type:"TITLE",payload : e.target.value})}></input>
+            <textarea className="full-width half-height text-s padding-s note-body" placeholder="Please Enter the Content of your Note" style = {{height : "10rem",font:"inherit"}} maxLength = "50" value = {body} onChange = {(e) => notesDispatch({type:"BODY",payload : e.target.value})}></textarea>
```

Comment:
I think it will be better to use controlled components for these two inputs instead of binding them to the context.